### DISCUSSION TITLE
Language Bar

### DISCUSSION THREAD

**Discussion by AssassinAguilar** [github-linguist/linguist#5387] [Q&A] [answered]
Please help me idk linguist-vendored. In my 1 repository there's no language bar as my files are in folders like dist, docs. I have made .gitattribute file and done some experiments but nothing happens I just want to add a language bar in my repo here's my repo https://github.com/AssassinAguilar/Alertism

**Comment by lildude** [comment, marked as answer]
Ooo, you've got a tricky one, and have a mistake. We'll start with the mistake: `docs` is considered documentation so you need to use the `linguist-documentation` override.

The tricky scenario is that your files under `dist` are considered _both_ vendored (because of the directory) and generated (because the content is minified) so you need to use _both_ `linguist-vendored` and `linguist-generated` overrides if you want them to count.

This will do the trick:

```gitattributes
dist/** -linguist-vendored -linguist-generated
docs/** -linguist-documentation
```

I've used recursive wildcards so you don't have to keep updating it as you add new files and directories. I've also used the syntax that is not Linguist-specific. It's the same as appending `=false` to each.

**Reply by AssassinAguilar** [reply]
WoooooooooooooW Thank you so much for the answer bro :D It works
